tests/tekton-tasks_test.go

```diff
@@ -242,4 +245,41 @@ var _ = Describe("Tekton-tasks", func() {
 			}, tenSecondTimeout, time.Second).Should(BeTrue(), "there should be no role bindings left")
 		})
 	})
+	Context("multiple CRs deployed", func() {
+		AfterEach(func() {
+			// To reset TTO we need to delete it and deploy it again
+			tto := strategy.GetTTO()
+			apiClient.Delete(ctx, tto)
+			strategy.CreateTTOIfNeeded()
+			waitUntilDeployed()
+		})
+
+		It("check if correct status is set for TTO", func() {
+			tto1 := strategy.createTekton("tto-test-1")
+			tto2 := strategy.createTekton("tto-test-2")
+
+			tektonTasksCRList := &tekton.TektonTasksList{}
+			tektonTasksCRList.SetGroupVersionKind(schema.GroupVersionKind{
+				Kind:    "TektonTasks",
+				Version: "tektontasks.kubevirt.io/v1alpha1",
+			})
+			apiClient.List(ctx, tektonTasksCRList)
+
+			for _, item := range tektonTasksCRList.Items {
+				for _, condition := range item.Status.Conditions {
+					if condition.Type == conditionsv1.ConditionAvailable {
+						Expect(condition.Status).To(Equal(v1.ConditionFalse))
+					}
+					if condition.Type == conditionsv1.ConditionProgressing {
+						Expect(condition.Status).To(Equal(v1.ConditionFalse))
+					}
+					if condition.Type == conditionsv1.ConditionDegraded {
+						Expect(condition.Status).To(Equal(v1.ConditionTrue))
+					}
+				}
+			}
+			apiClient.Delete(ctx, tto1)
```

Comment:
Move this into `AfterEach`?

Reply:
and please add check if deletion was successful